util/start_test

```diff
@@ -582,17 +584,24 @@ def check_environment():
             print("Error: Cannot find {0}.".format(util_dir))
             sys.exit(1)
 
+def check_environment_with_args():
     # find test dir and check for access
     global test_dir, auto_gen_spec_tests
-    test_dir = os.path.join(home, "test")
+
     auto_gen_spec_tests = os.access(os.path.join(home, "spec"), os.F_OK)
-    if (not os.access(test_dir, os.F_OK) or not os.access(test_dir, os.W_OK)
-        or not os.access(test_dir, os.X_OK)):
-        test_dir = os.path.join(home, "examples")
-        auto_gen_spec_tests = False
+
+    if args.test_root_dir and re.match(r'/+[^/]', str(args.test_root_dir)):
+        test_dir = str(args.test_root_dir)
```

Comment:
You are right, I should have kept exactly the same test on args.test_root_dir as the existing code already used (line 706 this file):
    # test root dir
    if args.test_root_dir:
        os.environ["CHPL_TEST_ROOT_DIR"] = str(args.test_root_dir)

I will just change line 593, to match the above.